Makefile

```diff
@@ -166,7 +166,7 @@ bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 deploy-setup: install
-	@EXCLUSIONS=() hack/deploy-setup.sh $(NAMESPACE)
+	hack/deploy-setup.sh $(NAMESPACE)
```

Comment:
@Mmduh-483 is it possible to fix the syntax rather remove it?

Reply:
changed to `@EXCLUSIONS= hack/deploy-setup.sh $(NAMESPACE)`